dart/README.md

```diff
@@ -5,13 +5,20 @@
   <br />
 </p>
 
-Sentry SDK for Dart and Flutter
+Sentry SDK for Dart
```

Comment:
So it looks smaller on pub.dev

![image](https://user-images.githubusercontent.com/1633368/99845837-4b3ca180-2b43-11eb-8548-e312958b7164.png)